guides/actions/item-actions.md

```diff
@@ -151,6 +153,9 @@ defmodule DemoWeb.ItemAction.SoftDelete do
     @impl Backpex.ItemAction
     def label(_assigns), do: Backpex.translate("Delete")
 
+    @impl Backpex.ItemAction
+    def confirm(_assigns), do: "Why do you want to delete this item?"
```

Comment:
The confirm function is mandatory.

Reply:
We could omit the `confirm_label` and `cancel_label`

Reply:
The `confirm/1` function determines whether a confirmation modal is shown to the user. We can decide to add the `confirm/1`, `confirm_label/1` and `cancel_label/1` functions (a modal is shown) or remove them (no modal is shown) 🤔.

If we add the `confirm/1` function as suggested, we must also keep `confirm_label/1` and `cancel_label/1`.

Reply:
The current example in the documentation makes no sense (`confirm_label/1` and `cancel_label/1` without modal).

> If we add the confirm/1 function as suggested, we must also keep `confirm_label/1` and `cancel_label/1`.

Thats not right. When omitted the default values ("Cancel" and "Apply") are used.

Reply:
I did not see the default implementation 🙃